sonar-vbnet-plugin/pom.xml

```diff
@@ -253,6 +253,9 @@
                         <include name="SonarAnalyzer.VisualBasic.dll"/>
                         <include name="Google.Protobuf.dll"/>
                       </fileset>
+                      <fileset dir="${project.build.directory}/../..">
+                        <include name="THIRD-PARTY-NOTICES.txt"/>
```

Comment:
File is missing on my machine in resulting jar file.

Reply:
replied in https://github.com/SonarSource/sonar-dotnet/pull/4662#discussion_r669447541